packages/tables/resources/views/components/row.blade.php

```diff
@@ -23,7 +23,7 @@
     @endif
     {{
         $attributes->class([
-            'fi-ta-row transition duration-75',
+            'fi-ta-row transition duration-0 md:duration-75',
```

Comment:
How about `[@media(hover:hover)]:duration-75` so it only affects non-touch devices? Also, do we still need `duration-0`?

Reply:
Unfortunately yes, otherwise the default duration from `transition` will be applied

Reply:
IMO we should apply transition with the media query then.

Reply:
Applied a couple fixes here, @danharrin @qrasa1: https://github.com/filamentphp/filament/commit/baf1cb6962976243d94816dada80a5b935d1a143.